packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -25,14 +25,21 @@ if (env['process.env.NODE_ENV'] !== '"production"') {
 }
 
 // We use "homepage" field to infer "public path" at which the app is served.
+// You may also use the environment variable CDN_URL to set the homepage to an absolute url
```

Comment:
This comment now looks a bit messy. Can you please format it so that sentences start with capital letters, end with newlines, and the complete comment doesn't look like it's written by two different people?